icons/bbc.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BBC</title><path d="M13.004 13c0 .744-.925.7-.925.7h-.925v-1.343h.925c.952-.007.925.644.925.644m-1.85-2.693h.704c.732.04.705.584.705.584 0 .677-.81.688-.81.688h-.6zm1.679 1.536s.633-.27.627-.985c0 0 .096-1.173-1.458-1.316h-1.724v4.917h1.977s1.65.004 1.65-1.388c0 0 .04-.947-1.072-1.228M8.37 8.58h7.258v6.84H8.371zM4.633 13c0 .744-.925.7-.925.7h-.925v-1.343h.925c.952-.007.925.644.925.644m-1.85-2.693h.705c.732.04.704.584.704.584 0 .677-.81.688-.81.688h-.599zm1.679 1.536s.633-.27.627-.985c0 0 .097-1.173-1.457-1.316H1.908v4.917h1.976s1.651.004 1.651-1.388c0 0 .04-.947-1.073-1.228M0 8.58h7.259v6.84H0zm22.52 1.316v.908s-.887-.545-1.867-.556c0 0-1.828-.036-1.91 1.752 0 0-.066 1.645 1.888 1.738 0 0 .82.099 1.932-.61v.94s-1.492.887-3.22.204c0 0-1.454-.53-1.509-2.272 0 0-.06-1.79 1.878-2.385 0 0 .517-.198 1.447-.11 0 0 .556.055 1.36.39m-5.778 5.525H24V8.58h-7.259Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BBC</title><path d="M0 8.57v6.86h6.86V8.57H0Zm1.92 1.54h1.46c.43 0 .76.08 1 .25.24.17.36.4.36.72a.9.9 0 0 1-.13.47.87.87 0 0 1-.38.31c.23.08.4.2.53.36.12.16.18.35.18.58 0 .23-.06.42-.18.58a1.1 1.1 0 0 1-.5.38 2.1 2.1 0 0 1-.79.13H1.92V10.1Zm.79.61v.94h.6c.2 0 .36-.04.46-.12.11-.1.17-.21.17-.37 0-.3-.21-.45-.63-.45h-.6Zm0 1.54v1.02h.69a1 1 0 0 0 .54-.13.43.43 0 0 0 .2-.38c0-.34-.25-.51-.73-.51h-.7Zm14.43-3.69v6.86H24V8.57h-6.86Zm4.3 1.53c.19.04.36.1.52.19v.75a1.78 1.78 0 0 0-1-.3c-.26 0-.48.05-.67.15-.18.1-.32.24-.42.42-.09.2-.14.42-.14.69 0 .26.05.5.14.68a1 1 0 0 0 .4.43c.19.1.4.15.66.15.38 0 .74-.1 1.07-.31v.73c-.15.1-.33.16-.54.2a2.57 2.57 0 0 1-1.45-.04 1.67 1.67 0 0 1-1-1.01 2.46 2.46 0 0 1 .01-1.64 1.73 1.73 0 0 1 1.02-1.02 2.41 2.41 0 0 1 1.4-.07ZM8.57 8.57v6.86h6.86V8.57H8.57Zm1.92 1.54h1.46c.43 0 .77.08 1 .25.24.17.36.4.36.72a.9.9 0 0 1-.13.47.87.87 0 0 1-.38.31c.23.08.4.2.53.36.12.16.18.35.18.58 0 .23-.06.42-.18.58a1.1 1.1 0 0 1-.5.38 2.1 2.1 0 0 1-.79.13H10.5V10.1Zm.8.61v.94h.59c.2 0 .36-.04.46-.12.11-.1.17-.21.17-.37 0-.3-.21-.45-.63-.45h-.6Zm0 1.54v1.02h.68c.24 0 .42-.05.54-.13a.43.43 0 0 0 .2-.38c0-.34-.25-.51-.73-.51h-.7Z"/></svg>
```

Comment:
There is an SVG baked into the BBC webpages which can be used.
Some minor differences compared to your icon - see website version in red.

![image](https://github.com/simple-icons/simple-icons/assets/1830867/c8c5b4da-1881-4d72-87b6-3be35a9a29c4)